cmd/lakectl/cmd/local_init.go

```diff
@@ -0,0 +1,83 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"github.com/treeverse/lakefs/pkg/git"
+	"github.com/treeverse/lakefs/pkg/uri"
+)
+
+const (
+	initMinArgs = 1
+	initMaxArgs = 2
+)
+
+var localInitCmd = &cobra.Command{
+	Use:   "init [directory] <lakeFS path URI>",
+	Short: "set a local directory to sync with a lakeFS ref and path",
+	Args:  cobra.RangeArgs(initMinArgs, initMaxArgs),
+	Run: func(cmd *cobra.Command, args []string) {
+		var remote *uri.URI
+		dir := "."
+		if len(args) == initMaxArgs {
+			dir = args[0]
+			remote = MustParsePathURI("remote ref", args[1])
+		} else {
+			remote = MustParsePathURI("remote ref", args[0])
+		}
```

Comment:
Think the first argument which is required should be the path uri to our repository